server/src/main/java/org/opensearch/gateway/remote/RemoteDownloadStats.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.gateway.remote;
+
+import org.opensearch.cluster.coordination.PersistedStateStats;
+
+public class RemoteDownloadStats extends PersistedStateStats {
+    static final String REMOTE_DOWNLOAD = "remote_download";
+
+    public RemoteDownloadStats() {
+        super(REMOTE_DOWNLOAD);
+    }
+}
```

Comment:
Why do we need a new class?

Reply:
RemoteDownloadStats extends the PersistedStateStats which stores the stats for different fields like success_count, failed_count. We needed a new composite type `remoteDownload` so that we can have these fields for download stats. I have created a class so that we can add more stats for download as need arise similar to what we have done for RemoteUploads.

Reply:
If we don't have a concrete use case now can we start with just an instance?